nuxt.config.js

```diff
@@ -1,61 +1,104 @@
-      {rel: 'icon', type: 'image/png', sizes: '16x16', href: '/icons/favicon-16x16.png'},
-      {rel: 'icon', type: 'manifest', href: '/icons/site.webmanifest'},
-      {rel: 'mask-icon', href: '/icons/safari-pinned-tab.svg', color: '#5bbad5'},
-      {rel: 'shortcut icon', href: '/icons/favicon.ico'},
-      {rel: 'icon', type: 'image/x-icon', href: '/icons/favicon.ico'},
-      {rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons'},
-      {rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Lato'},
-      {
-        rel: 'stylesheet',
-        href: 'https://use.fontawesome.com/releases/v5.0.10/css/all.css',
-        integrity: 'sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg',
-        crossorigin: 'anonymous'
+      {
+        rel: "apple-touch-icon",
+        sizes: "180x180",
+        href: "/icons/apple-touch-icon.png"
+      },
+      {
+        rel: "icon",
+        type: "image/png",
+        sizes: "32x32",
+        href: "/icons/favicon-32x32.png"
+      },
+      {
+        rel: "icon",
+        type: "image/png",
+        sizes: "16x16",
+        href: "/icons/favicon-16x16.png"
+      },
+      { rel: "icon", type: "manifest", href: "/icons/site.webmanifest" },
+      {
+        rel: "mask-icon",
+        href: "/icons/safari-pinned-tab.svg",
+        color: "#5bbad5"
+      },
+      { rel: "shortcut icon", href: "/icons/favicon.ico" },
+      { rel: "icon", type: "image/x-icon", href: "/icons/favicon.ico" },
+      {
+        rel: "stylesheet",
+        href:
+          "https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
+      },
+      {
+        rel: "stylesheet",
+        href: "https://fonts.googleapis.com/css?family=Lato"
+      },
+      {
+        rel: "stylesheet",
+        href: "https://use.fontawesome.com/releases/v5.0.10/css/all.css",
+        integrity:
+          "sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg",
+        crossorigin: "anonymous"
       }
     ]
   },
   plugins: [
-    '~/plugins/vuetify',
-    '~/plugins/eventBus.js',
-    '~/plugins/i18n.js',
-    '~/plugins/vue-moment.js'
+    "~/plugins/vuetify",
+    "~/plugins/eventBus.js",
+    "~/plugins/i18n.js",
+    "~/plugins/vue-moment.js"
   ],
   css: [
-    '~/assets/css/app.styl',
-    '~/assets/css/main.scss',
-    'flag-icon-css/css/flag-icon.min.css',
+    "~/assets/css/app.styl",
+    "~/assets/css/main.scss",
+    "flag-icon-css/css/flag-icon.min.css"
   ],
   modules: [
-    ['@nuxtjs/google-analytics', {
-      id: 'UA-65309624-3'
-    }],
-    ['storyblok-nuxt', {accessToken: 'plc0fBh2no8owEeTALSN2wtt', cacheProvider: 'memory'}],
+    [
+      "@nuxtjs/google-analytics",
+      {
+        id: "UA-65309624-3"
+      }
+    ],
+    [
+      "storyblok-nuxt",
+      { accessToken: "plc0fBh2no8owEeTALSN2wtt", cacheProvider: "memory" }
+    ],
+    "@nuxtjs/axios",
+    "@nuxtjs/proxy"
```

Comment:
**praise**: Great work utilizing Nuxt Proxy module for fighting CORS! 🎉

Reply:
Thanks :)